frontend/src/features/workspace/components/CreateFormModal/CreateFormModalContent/FormResponseOptions.tsx

```diff
@@ -70,13 +73,9 @@ export const FormResponseOptions = forwardRef<
           <Tile.Subtitle>{storage.subtitle}</Tile.Subtitle>
           <OptionDescription
             listItems={[
-              { text: 'Supports email submissions' },
-              { text: 'Supports Singpass & Myinfo' },
-              { text: 'Supports webhooks for integrations' },
-              {
-                text: 'Up to Confidential (Cloud-Eligible) and Sensitive (High) data',
-                badge: 'new',
-              },
```

Comment:
This has gone missing, probably just because the PR has been around for a while. I'll try to add it later, unless you get to it first.